src/modules/75-cf/components/TableFilters/TableFilters.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import { Layout } from '@harness/uicore'
+import type { StringKeys } from 'framework/strings'
+import { FilterCard } from './FilterCard'
+
+interface TableFiltersProps {
+  filters: Array<{
+    labelStringKey: StringKeys
+    total: number
+    queryProps: { key: string; value: string | boolean | number }
+    tooltipId?: string
+  }>
+  currentFilter: { key: string; value: string | boolean | number } | false
+  updateTableFilter(queryProps: { key: string; value: string | boolean | number } | false): void
```

Comment:
A more common way of typing a function coming in as an argument/prop is
```suggestion
  updateTableFilter: (queryProps: { key: string; value: string | boolean | number } | false) => void
```

That way it keeps the name of the prop and the definition of the function distinct

Reply:
Yeah, that's exactly it. They both work (I think there's actually a third way that Typescript supports, but it escapes me), but the `key: prop` is just a bit more obvious to scan for my eyes anyway